stores/sql/main.go

```diff
@@ -1843,22 +1851,63 @@ func Slab(ctx context.Context, tx sql.Tx, key object.EncryptionKey) (object.Slab
 	}
 	defer stmt.Close()
 
+	// fetch hosts for each sector
+	hostStmt, err := tx.Prepare(ctx, `
```

Comment:
This causes the `Contracts` to contain empty slices for hosts that have a `host_sectors` link but no `contract_sectors` link.